.github/workflows/perf_nightly.yml

```diff
@@ -0,0 +1,165 @@
+name: Nightly Benchmarks on Nyrkiö Runners (stability)
+
+on:
+  workflow_dispatch:
+    branches: ["main", "notmain", "master"]
+  schedule:
+    - cron: '24 6,11,14,20 * * *'
+  push:
+  #   branches: ["main", "notmain", "master"]
+    branches: ["notmain"]
+  pull_request:
+  #   branches: ["main", "notmain", "master"]
+    branches: ["notmain"]
+
+env:
+  CARGO_TERM_COLOR: never
+
+jobs:
+  bench:
+    runs-on: nyrkio_perf_server_2cpu_ubuntu2404
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+      - uses: useblacksmith/setup-node@v5
+        with:
+          node-version: 20
+      #     cache: 'npm'
+      # - name: Install dependencies
+      #   run: npm install && npm run build
+
+      - name: Bench
+        run: make bench-exclude-tpc-h  2>&1 | tee output.txt
+      - name: Analyze benchmark result with Nyrkiö
+        uses: nyrkio/change-detection@HEAD
+        with:
+          name: nightly/turso
```

Comment:
This is copy paste of the existing rust_perf.yml, just changed prefix of the test names.